app/app.go

```diff
@@ -177,30 +184,25 @@ type GaiaApp struct {
 }
 
 func init() {
-	userHomeDir, err := os.UserHomeDir()
-	if err != nil {
-		panic(err)
-	}
-
+	userHomeDir := os.ExpandEnv("$HOME")
```

Comment:
`os.UserHomeDir()` ensures better portability, you may see its implementation here: https://golang.org/src/os/file.go?s=14399:14433#L479

Reply:
os.UserHomeDir() panics in some environments like serverless environments like AWS lamba.

A number of users requested using the environment variable so gaia can be used as a library.

Open to other ways of doing this

Reply:
This makes sense. Please feel free to ignore my comment :+1: